EasyLearn/app/assets/stylesheets/player.css

```diff
@@ -0,0 +1,93 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;500&display=swap');
+
+nav {
+  background-color:#282828;
```

Comment:
I recommend you sort your properties of your styles alphabetically, It is considered good practice :smile:

Reply:
Sorted

Reply:
I do not see that it is sorted